src/core/toga/widgets/numberinput.py

```diff
@@ -20,16 +20,35 @@ class NumberInput(Widget):
     MIN_WIDTH = 100
 
     def __init__(self, id=None, style=None, factory=None, step=1,
-                 min_value=None, max_value=None, readonly=False, on_change=None):
+                 min_value=None, max_value=None, readonly=False, on_change=None,
+                 adjustment=None):
         super().__init__(id=id, style=style, factory=factory)
         self._value = None
         self._impl = self.factory.NumberInput(interface=self)
 
-        self.readonly = readonly
-        self.step = step
         self.min_value = min_value
         self.max_value = max_value
         self.on_change = on_change
+        self.readonly = readonly
+        self.step = step
+        self.adjustment = adjustment
+
+    
+    @property
+    def adjustment(self):
+        """ Adjustment for the numberinput widget
+
+        Returns:
+            Adjustment value for the widget
+        """
+        return self._adjustment
+
+    @adjustment.setter
+    def adjustment(self, value):
+        try:
+            self._impl.set_adjustment_value()
+        except (ValueError, TypeError):
+            raise ValueError("Adjustment Could Not be set")
```

Comment:
Capitalization is a little odd here

Reply:
Didn't even realize this - definitely looks odd! 😅 Changing that in the amend commit